libs/vm_monitor/src/cgroup.rs

```diff
@@ -1,28 +1,26 @@
 use std::{
     fmt::{Debug, Display},
     fs,
-    pin::Pin,
+    pin::pin,
     sync::atomic::{AtomicU64, Ordering},
-    task::{self, Poll},
 };
 
-use anyhow::{anyhow, Context};
-use async_std::channel::{Receiver, Sender};
+use anyhow::{anyhow, bail, Context};
 use cgroups_rs::{
     freezer::FreezerController,
     hierarchies::{self, is_cgroup2_unified_mode, UNIFIED_MOUNTPOINT},
     memory::MemController,
     MaxValue,
     Subsystem::{Freezer, Mem},
 };
-use futures::{stream::BoxStream, Future, FutureExt, Stream};
 use inotify::{EventStream, Inotify, WatchMask};
+use tokio::sync::mpsc::{error::TryRecvError, Receiver, Sender};
```

Comment:
NIT: because there's many different `Receiver`/`Sender` types, I always reference them as `mpsc::Receiver` or `mpsc::Sender`, so make it easier to understand the code without needing to read the imports.

cc @koivunej in case I'm crazy for this

Reply:
Agreed.